man/org.freedesktop.systemd1.xml

```diff
@@ -1171,6 +1174,11 @@ node /org/freedesktop/systemd1 {
       the "Try" flavor is used in which case a service that isn't running is not affected by the restart. The
       "ReloadOrRestart" flavors attempt a reload if the unit supports it and use a restart otherwise.</para>
 
+      <para><function>StartAutomaticJobs()</function> creates reload/restart jobs for units which have been
```

Comment:
hmm, thing is though, these aren't automatic at all, they require manual triggering after all ;-)

hmm, we also usually start units, not jobs. The jobs are what starting of a unit results in...

also, these don't actually start anything, they reload and restart — only if already running

maybe call this EnqueueMarkedJobs() or so?